DEVELOPMENT.md

```diff
@@ -202,3 +202,35 @@ Additional information which is really important to how the application runs can
 be found in the manifest:
 
 > `$ bundle exec 3scale_backend manifest`
+
+## Running with RubyMine
+
+Using RubyMine scripts can help you run both Worker and Listener.
+
+For that, you'll need to create the following configurations: 
+
+### Worker
+
+| Name                  | `worker` |
```

Comment:
You would need to add a header for the table to make it look good. I tried this too at first (because there is no real need for the header), but it just doesn't work :grimacing: 

![image](https://github.com/3scale/apisonator/assets/1270328/4fd3cce5-445b-4417-bb81-5d28fd317d01)